### PR TITLE
Add fetching number of sessions on web backends to haproxy module

### PR DESCRIPTION
**Pull request by mchugh19**
### What does this PR do?

Adds fetching the number of current sessions (scur) for web backends to the haproxy module
Also adds a haproxy beacon allowing for triggering events when scur is over a specified threshold
### Tests written?

No

### CODE REVIEW

**Comment by s0undt3ch** [comment]
For the version, use the name if the next major release. Look it up in `salt.version`. Then its not a guess :)

File: salt/beacons/haproxy.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Watch current connections of haproxy server backends.
 Fire an event when over a specified threshold.
 
-.. versionadded:: 2016.9.0
+.. versionadded:: Carbon
```
Comment (mchugh19) [inline salt/beacons/haproxy.py]: Done

File: salt/modules/haproxyconn.py [hunk 2]
```diff
@@ -256,3 +256,36 @@ def show_backends(socket='/var/run/haproxy.sock'):
     ha_conn = _get_conn(socket)
     ha_cmd = haproxy.cmds.showBackends()
     return ha_conn.sendCmd(ha_cmd)
+
+def get_sessions(name, backend, socket='/var/run/haproxy.sock'):
+    '''
+    Get number of current session on server in backend (scurr)
+
```
Comment (rallytime) [inline salt/modules/haproxyconn.py]: This new function could use a `.. versionadded:: Carbon` directive here.

**Comment by rallytime** [comment]
@mchugh19 Thanks for adding this. I had one small comment above, and there are some [lint violations](http://166.78.178.63:8080/job/PR/job/salt-pr-lint-n/1314/violations/) that need to be addressed. Once those are fixed up, we can get this in.

**Comment by rallytime** [comment]
Thanks @mchugh19!

**Pull request merged**